Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -53,7 +53,7 @@ extension LogtoCore {
         codeVerifier: String,
         tokenEndpoint: String,
         clientId: String,
-        resource: String? = nil,
+        resources: String? = nil,
```

Comment:
since two fetchToken functions just require one resource, no need to rename here?
https://www.notion.so/silverhand/fetchTokenByRefreshToken-a1bc0fd7d79a4ebe9fd4c9168ae4a4d8
https://www.notion.so/silverhand/fetchTokenByAuthorizationCode-13295cb7e1af4f4ea6d4c23c003ea7b7

Reply:
the code request accepts multiple resources

Reply:
updated, ptal

Reply:
Then why not using `[string]` here as well?

Reply:
> the code request accepts multiple resources

only one `resource` is allowed and in the signIn method, we don't need to add this query param.

* [Slack Discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1642737298120800?thread_ts=1642735239.103400&cid=C02A8G4HVAM)
* [Notion Doc](https://www.notion.so/silverhand/fetchTokenByAuthorizationCode-13295cb7e1af4f4ea6d4c23c003ea7b7#3e6db81e0d944241af55c1cc662407f0)
![image](https://user-images.githubusercontent.com/10806653/151692298-b896ea75-c075-4bed-92e9-3f0c23112c07.png)

Reply:
Conclusion: No need to rename here.

Reply:
updated